src/components/submissionPage/SubmissionTextBox.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import Snap from '../../firebase/snap/Snap';
+import SubmitSnap from '../../firebase/snap/SubmitSnap';
+
+
+const SubmissionTextBox:React.FunctionComponent = () => {
+  /* Containing body of the snap */
+  const [message, setMessage] = useState<string>("");
+  const [confirmation, setConfirmation] = useState<Boolean>(false);
+  const [error,setError] = useState<string>("");
+  
+  function handleSubmit(event){
+    //TODO: Get auth().userID
+    event.preventDefault();
+    const resultingSnap : Snap = {
+      to : [""], 
+      from: "", 
+      body: message, 
+      timestamp: new Date()
+    };
+    const res = SubmitSnap(resultingSnap, setError, setConfirmation).then(
+      () => {
+        setMessage("");
+      }
+    );
+  }
+
+  /* Updates the value in the webhook "message" */
+  function handleChange(event){
```

Comment:
```suggestion
  function onMessageTextChanged(event){
```
A more descriptive name removes the need for the comment.

Reply:
has been changed, here and in the other places this name is used